build-tools/xaprepare/xaprepare/xaprepare.csproj

```diff
@@ -8,7 +8,7 @@
     <RootNamespace>Xamarin.Android.Prepare</RootNamespace>
     <AssemblyName>xaprepare</AssemblyName>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <LangVersion>latest</LangVersion>
+    <LangVersion>preview</LangVersion>
```

Comment:
If this needs to change, it should probably be changed to `8.0`, not `preview`.

Reply:
8.0 it is then. b5221c9a5f24e8d612b2379ac33bcff3c9949253